encrypted-media-respec.html

```diff
@@ -1598,6 +1591,10 @@
         The above rule implies that the CDM instance must not be destroyed until all <a>MediaKeys</a>
         objects and all <a>MediaKeySession</a> objects associated with the CDM instance are destroyed.
       </p>
+      <p>
+        When a <a>MediaKeySession</a> object is destroyed, the CDM SHALL close the <a href="#key-session">key session</a> associated with the object.
```

Comment:
I don't think that would accurately reflect any implementation, though. Nor do I think authors should rely on this. I've added a non-normative note to this effect.

Garbage collection seems to be the best platform concept we have for such things. For example, see https://html.spec.whatwg.org/multipage/embedded-content.html#best-practices-for-authors-using-media-elements. (We could replace "destroyed" with "garbage collection occurs") Also note that it is recommended that authors explicitly call `load()` rather than waiting for garbage collection.

Other than potentially replacing "destroyed," I believe the current commit (a60a130) is consistent with how HTML5 deals with similar resource release issues.